components/director/internal/graphql/scalars/labels_test.go

```diff
@@ -0,0 +1,43 @@
+package scalars
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestLabels_UnmarshalGQL(t *testing.T) {
+	//given
```

Comment:
Yes, error case could be also good to test for all scalars

Reply:
done